### ISSUE TITLE
Add Toolbar and Navigation layout on top of Screens with this template

### ISSUE THREAD

**Issue by YarosMallorca** [kulothunganug/kivy-lazy-loading-template#1]
Hello,
We have an app with several screens, MDToolbar and MDNavigationLayout.
How can we use this template, load the screens below the Toolbar and Navigation Layout?
This is the current KV code:

```
#:import HomeScreen Screens.baseclass.homescreen.HomeScreen
#:import Area Screens.baseclass.area.Area
#:import Angle Screens.baseclass.angle.Angle
#:import Calculator Screens.baseclass.calculator.Calculator
#:import Currency Screens.baseclass.currency.Currency
#:import DataTransferRate Screens.baseclass.datatransferrate.DataTransferRate
#:import DigitalStorage Screens.baseclass.digitalstorage.DigitalStorage
#:import Energy Screens.baseclass.energy.Energy
#:import Frequency Screens.baseclass.frequency.Frequency
#:import Length Screens.baseclass.length.Length
#:import Mass Screens.baseclass.mass.Mass
#:import Pressure Screens.baseclass.pressure.Pressure
#:import Speed Screens.baseclass.speed.Speed
#:import Temperature Screens.baseclass.temperature.Temperature
#:import Time Screens.baseclass.time.Time
#:import Volume Screens.baseclass.volume.Volume
    
Screen:
    FloatLayout:
        MDToolbar:
            pos_hint: {"top": 1}
            elevation: 10
            left_action_items: [['menu', lambda x: nav_drawer.set_state("open")]]
            right_action_items: [["palette", lambda x: app.change_theme()], ["information", lambda x: app.about_us()]]
            
    MDNavigationLayout:
        ScreenManager:
            id: screen_manager
            
            HomeScreen:
                name: "homescreen"
                
            Area:
                name: "area"
                
            Angle:
                name: "angle"
                
            Calculator:
                name: "calculator"
                
            Currency:
                name: "currency"
                
            DataTransferRate:
                name: "datatransferrate"
                
            DigitalStorage:
                name: "digitalstorage"
                
            Energy:
                name: "energy"
                
            Frequency:
                name: "frequency"
                
            Length:
                name: "length"
                
            Mass:
                name: "mass"
                
            Pressure:
                name: "pressure"
                
            Speed:
                name: "speed"
                
            Temperature:
                name: "temperature"
                
            Time:
                name: "time"
                
            Volume:
                name: "volume"
                
            

        MDNavigationDrawer:
            id: nav_drawer
            close_on_click: True

            ContentNavigationDrawer:
                id: navdrawer
                screen_manager: screen_manager
                nav_drawer: nav_drawer
```

Any help would be appreciated!
Thanks!

**Comment by kulothunganug** [comment]
Hi @YarostheLaunchpadder ,
To show the `Toolbar` in all screens, you need to make the `Root` class as `BoxLayout`.
And use `Builder.load_kv` in `root.py`, in that `BoxLayout`, add the `Toolbar` and `ScreenManager` respectively.
You should also need to give a `id` to `ScreenManager` and use `self.ids.scr_mgr_id.add_widget` instead of  `self.add_widget`, `self.ids.scr_mgr_id.has_screen` instead of `self.has_screen`, in `Root.set_current` method.

In short
```py
Builder.load_kv("""
<Root>
    orientation: 'vertical'
    
    Toolbar:
    ScreenManager:
        id: scr_mgr
""")

class Root(BoxLayout):
    def set_current(self, screen_name, side='left'):
        # ...
        if  not self.ids.scr_mgr.has_screen(screen_name):
            # ...
            self.ids.scr_mgr.add_widget(screen_object)
        # ...
```
Still I did't tested it, Hope this will work 🙂

**Comment by YarosMallorca** [comment]
Thank you very much! Will try it!
